### ISSUE TITLE
Redirect to client fail after two step verification done

### ISSUE THREAD

**Issue by neerajyadav** [IdentityServer/IdentityServer4.Samples#102]
After confirming security code on two step verification view, it does not redirect to client.

[Here](https://github.com/IdentityServer/IdentityServer4.Samples/blob/dev/Quickstarts/6_AspNetIdentity/src/IdentityServerWithAspNetIdentity/Views/Account/VerifyCode.cshtml#L8) returnurl should be passed as `asp-route-returnurl="@Model.ReturnUrl"` instead of `asp-route-returnurl="@ViewData["ReturnUrl"]"`

**Comment by brockallen** [comment]
PR?

**Comment by neerajyadav** [comment]
Yes Sure.. Will do that. 
I'm also facing another issue with rendering external css and scripts after applying `[SecurityHeader]` ([here](https://github.com/IdentityServer/IdentityServer4.Samples/blob/dev/Quickstarts/6_AspNetIdentity/src/IdentityServerWithAspNetIdentity/Filters/SecurityHeadersAttribute.cs))which you did in commit [update to rc5](https://github.com/IdentityServer/IdentityServer4.Samples/commit/35c9af86241b22297ad3726ccefa9431ab13cc42).

Locally it is working fine. After publishing the host, account pages are rendering find in IE, however external css and js files are blocked on chrome. Here is the log from browser console.

```
Refused to load the stylesheet 'https://fonts.googleapis.com/css?family=Roboto:regular,bold,italic,thin,light,bolditalic,black,medium&lang=en' because it violates the following Content Security Policy directive: "default-src 'self'". Note that 'style-src' was not explicitly set, so 'default-src' is used as a fallback.

login:11 Refused to load the stylesheet 'https://fonts.googleapis.com/icon?family=Material+Icons' because it violates the following Content Security Policy directive: "default-src 'self'". Note that 'style-src' was not explicitly set, so 'default-src' is used as a fallback.

login:12 Refused to load the stylesheet 'https://code.getmdl.io/1.2.1/material.purple-amber.min.css' because it violates the following Content Security Policy directive: "default-src 'self'". Note that 'style-src' was not explicitly set, so 'default-src' is used as a fallback.

login:28 Refused to apply inline style because it violates the following Content Security Policy directive: "default-src 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-aqNNdDLnnrDOnTNdkJpYlAxKVJtLt9CtFLklmInuUAE='), or a nonce ('nonce-...') is required to enable inline execution. Note also that 'style-src' was not explicitly set, so 'default-src' is used as a fallback.

login:1 Refused to load the script 'https://ajax.aspnetcdn.com/ajax/jquery/jquery-2.2.0.min.js' because it violates the following Content Security Policy directive: "default-src 'self'". Note that 'script-src' was not explicitly set, so 'default-src' is used as a fallback.

login:99 Refused to execute inline script because it violates the following Content Security Policy directive: "default-src 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-5JpMkUcnuO2LhocOq2EVe2RTjZN6AS1M3w2+xVNahH8='), or a nonce ('nonce-...') is required to enable inline execution. Note also that 'script-src' was not explicitly set, so 'default-src' is used as a fallback.

login:1 Refused to load the script 'https://code.getmdl.io/1.2.1/material.min.js' because it violates the following Content Security Policy directive: "default-src 'self'". Note that 'script-src' was not explicitly set, so 'default-src' is used as a fallback.

login:102 Refused to execute inline script because it violates the following Content Security Policy directive: "default-src 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-Mm+rK7KdAfN+ySS/SAO0x5Q0LI0XyfGweLOiVd3FMAc='), or a nonce ('nonce-...') is required to enable inline execution. Note also that 'script-src' was not explicitly set, so 'default-src' is used as a fallback.

login:1 Refused to load the script 'https://ajax.aspnetcdn.com/ajax/jquery.validate/1.14.0/jquery.validate.min.js' because it violates the following Content Security Policy directive: "default-src 'self'". Note that 'script-src' was not explicitly set, so 'default-src' is used as a fallback.

login:111 Refused to execute inline script because it violates the following Content Security Policy directive: "default-src 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-Bjw0tbM5vAaD9soptLO7Hs66rxxskisVs/9rvAiSEl0='), or a nonce ('nonce-...') is required to enable inline execution. Note also that 'script-src' was not explicitly set, so 'default-src' is used as a fallback.

login:1 Refused to load the script 'https://ajax.aspnetcdn.com/ajax/jquery.validation.unobtrusive/3.2.6/jquery.validate.unobtrusive.min.js' because it violates the following Content Security Policy directive: "default-src 'self'". Note that 'script-src' was not explicitly set, so 'default-src' is used as a fallback.  
```

**Comment by brockallen** [comment]
Yea, you'll have to configure CSP. Keep in mind -- the quickstarts are samples. It's your responsibility to build your own app.
